kpops/cli/main.py

```diff
@@ -207,17 +217,22 @@ def log_action(action: str, pipeline_component: PipelineComponent):
 
 
 def create_kpops_config(
-    config: Path, defaults: Optional[Path], verbose: bool, dotenv: Optional[list[Path]]
+    config: Path,
+    defaults: Optional[Path],
+    verbose: bool,
+    dotenv: Optional[list[Path]],
+    environment: Optional[str],
 ) -> KpopsConfig:
     setup_logging_level(verbose)
-    YamlConfigSettingsSource.path_to_config = config
+    YamlConfigSettingsSource.path_to_settings = config
```

Comment:
I like this name more
```suggestion
    YamlConfigSettingsSource.path_to_config = config
```